tests/js/rps.scrypttest.js

```diff
@@ -0,0 +1,171 @@
+const { expect } = require("chai");
+const { bsv, buildContractClass, signTx, toHex, getPreimage, num2bin, PubKey, Ripemd160, SigHashPreimage, Sig } = require("scryptlib");
+const { inputIndex, inputSatoshis, newTx, compileContract, DataLen, dummyTxId } = require("../../helper");
+
+// make a copy since it will be mutated
+var tx_ = newTx();
```

Comment:
Let's rename all `tx_` to `tx`, as in other examples.

Reply:
got it.